aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,190 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
+   + <script src="_framework/blazor.web.js"></script>
+   ```
 
-> [!IMPORTANT]
-> The `@bind:get` and `@bind:set` features are receiving further updates at this time. To take advantage of the latest updates, confirm that you've installed the [latest SDK](https://dotnet.microsoft.com/download/dotnet/7.0).
->
-> Using an event callback parameter (`[Parameter] public EventCallback<string> ValueChanged { get; set; }`) isn't supported. Instead, pass an <xref:System.Action>-returning or <xref:System.Threading.Tasks.Task>-returning method to `@bind:set`.
->
-> For more information, see the following resources:
->
-> * [Blazor `@bind:after` not working on .NET 7 RTM release (dotnet/aspnetcore #44957)](https://github.com/dotnet/aspnetcore/issues/44957)
-> * [`BindGetSetAfter701` sample app (javiercn/BindGetSetAfter701 GitHub repository)](https://github.com/javiercn/BindGetSetAfter701)
+1. Update `Program.cs`:
 
-For more information, see the following content in the *Data binding* article:
+   > [!NOTE]
+   > In the following example, the app's namespace is `Net8BlazorServer`.
 
-* [Introduction](xref:blazor/components/data-binding?view=aspnetcore-7.0)
-* [Bind across more than two components](xref:blazor/components/data-binding?view=aspnetcore-7.0#bind-across-more-than-two-components)
+   Add a `using` statement for the app's namespace:
 
--->
+   ```csharp
+   using Net8BlazorServer;
+   ```
+  
+   Replace `AddServerSideBlazor` with `AddRazorComponents` and a chained call to `AddInteractiveServerComponents`.
+   
+   Remove the following line:
+
+   ```diff
+   - builder.Services.AddServerSideBlazor();
+   ```
+
+   Replace it with the following:
+
+   ```csharp
+   builder.Services.AddRazorComponents()
+       .AddInteractiveServerComponents();
+   ```
+
+   Replace `MapBlazorHub` with `MapRazorComponents`, supplying the `App` component as the root component type, and a chained call to `AddInteractiveServerRenderMode`.
+   
+   Remove the following line:
+  
+   ```diff
+   - app.MapBlazorHub();
+   ```
+
+   Replace it with the following:
+
+   ```csharp
+   app.MapRazorComponents<App>()
+       .AddInteractiveServerRenderMode();
+   ```
+
+   Remove the endpoint mapping that falls back to the `_Host` page:
+   
+   ```diff
+   - app.MapFallbackToPage("/_Host");
+   ```
+
+1. If the Blazor Server app wasn't prerendering content, you can continue to disable prerendering for the updated app. In the `App` component, change the value assigned to the `@rendermode` Razor directive attributes for the `HeadOutlet` and `Routes` components:
+
+   ```diff
+   - @rendermode="RenderMode.InteractiveServer"
+   + @rendermode="new InteractiveServerRenderMode(prerender: false)"
+   ```
+
+   For more information, see <xref:blazor/components/render-modes?view=aspnetcore-8.0&preserve-view=true#prerendering>.
+
+### Add Auto component rendering to a Blazor Server app
+
+
+
+### Upgrade a hosted Blazor WebAssembly app
+
+
+
+### Adopt .NET 8 features
```

Comment:
For you folks to go SO in-depth on migrating other apps, and then just say "create a new app and copy the code over" for WebAssembly apps feels like a REAL cop-out. I hope that's not the final guidance.

Reply:
It's TBD. Let's not discuss it at this point because this is a WIP. I've only just started work here.

Reply:
I'll get back to you on it.

Reply:
BTW tho ... this has (almost) always been our guidance for Blazor due to the high degree of template churn. I did write it all out at .NET 5, and it was real ***murder***. It took days to write it all out, then it took several trips back into the guidance to fix problems and call out gotchas. We still didn't cover auth updates, as they were even more extensive and complex.